### ISSUE TITLE
Side panel should gain focus when opened w/keyboard

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#8992]
The side panel does not have focus when a user using a keyboard as input modality opens it via the "Filter" button on the Library page (or the "Folders" button on the Topics page). The keyboard focus is still on the main content underneath the side panel.

Impact:
User is unable to use the filters in the side panel in "Filter"
User is unable to access folders in the side panel in "Folders" (on the Topics page)

![keyboard-navigation-sidebar](https://user-images.githubusercontent.com/1457929/148900693-d8a3ab9a-9df4-4631-87c8-359cdb5388a1.gif)

_Originally posted by @radinamatic in https://github.com/learningequality/kolibri/issues/8986#issuecomment-1009674923_
